License headers:

tools/testing/selftests/futex/functional/.gitignore:
# SPDX-License-Identifier: GPL-2.0-only

tools/testing/selftests/futex/functional/Makefile:
# SPDX-License-Identifier: GPL-2.0

tools/testing/selftests/futex/functional/run.sh:
# SPDX-License-Identifier: GPL-2.0-or-later

--- a/tools/testing/selftests/futex/functional/.gitignore.orig	2021-09-22 21:29:21.694921100 -0700
+++ b/tools/testing/selftests/futex/functional/.gitignore	2021-09-22 21:29:20.158921049 -0700
@@ -8,3 +8,4 @@ futex_wait_uninitialized_heap
 futex_wait_wouldblock
 futex_wait
 futex_requeue
+futex2_wait
--- a/tools/testing/selftests/futex/functional/Makefile.orig	2021-09-22 21:30:01.308922405 -0700
+++ b/tools/testing/selftests/futex/functional/Makefile	2021-09-22 21:30:03.132922465 -0700
@@ -18,7 +18,8 @@ TEST_GEN_FILES := \
 	futex_wait_uninitialized_heap \
 	futex_wait_private_mapped_file \
 	futex_wait \
-	futex_requeue
+	futex_requeue \
+	futex2_wait
 
 TEST_PROGS := run.sh
 
--- a/tools/testing/selftests/futex/functional/run.sh.orig	2021-09-22 21:28:55.711920244 -0700
+++ b/tools/testing/selftests/futex/functional/run.sh	2021-09-22 21:28:53.025920155 -0700
@@ -79,3 +79,6 @@ echo
 
 echo
 ./futex_requeue $COLOR
+
+echo
+./futex2_wait $COLOR
